js/securesubmit/checkout-form.js

```diff
@@ -313,17 +314,155 @@ document.observe('dom:loaded', function () {
         });
     }
 
-    // MageStore One Step Checkout
-    if (typeof oscPlaceOrder == 'function') {
-        var cloneFunction = function (that) {
-            var temp = function temporary() { return that.apply(this, arguments); };
-            for (var key in this) {
-                if (this.hasOwnProperty(key)) {
-                    temp[key] = this[key];
+
+    var cloneFunction = function (that) {
+        var temp = function temporary() {
+            return that.apply(this, arguments);
+        };
+        for (var key in this) {
+            if (this.hasOwnProperty(key)) {
+                temp[key] = this[key];
+            }
+        }
+        return temp;
+    };
+    // Amasty completeCheckout();
+
+    if (typeof completeCheckout == 'function' && document.querySelector('#amscheckout-onepage')) {
+        amastyCompleteCheckoutOriginal = cloneFunction(completeCheckout);
+
+        try {
+            var ele;
+            ele = document.createElement('div');
+            ele.id = 'co-payment-form-update';
+            var pEle = document.querySelector('#amscheckout-main > div.amscheckout > div > div.second-column > div:nth-child(3) > div.payment-method');
+            pEle.insertBefore(ele, pEle.childNodes[2]);
+        } catch (e) {
+        }
+        try {
+            document.querySelector('#payment-buttons-container').remove();
+        } catch (e) {
+        }
```

Comment:
@ttnnkkrr Does this need to be in a separate `try`/`catch`? If not, can you merge both of the above?